ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -254,31 +254,23 @@ objects:
         value: ${CLUSTER_TYPE}
       - name: AWS_SHARED_CREDENTIALS_FILE
         value: /etc/openshift-installer/.awscred
-      - name: OPENSHIFT_INSTALL_CLUSTER_NAME
+      - name: AWS_REGION
+        value: us-east-1
+      - name: CLUSTER_NAME
         value: ${NAMESPACE}-${JOB_NAME_HASH}
-      - name: OPENSHIFT_INSTALL_BASE_DOMAIN
+      - name: BASE_DOMAIN
         value: origin-ci-int-aws.dev.rhcloud.com
-      - name: OPENSHIFT_INSTALL_EMAIL_ADDRESS
-        value: test@ci.openshift.io
-      - name: OPENSHIFT_INSTALL_PASSWORD
-        value: verysecure
-      - name: OPENSHIFT_INSTALL_SSH_PUB_KEY_PATH
+      - name: SSH_PUB_KEY_PATH
         value: /etc/openshift-installer/ssh-publickey
-      - name: OPENSHIFT_INSTALL_PULL_SECRET_PATH
+      - name: PULL_SECRET_PATH
         value: /etc/openshift-installer/pull-secret
-      - name: OPENSHIFT_INSTALL_PLATFORM
-        value: ${CLUSTER_TYPE}
-      - name: OPENSHIFT_INSTALL_AWS_REGION
-        value: us-east-1
       - name: OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE
```

Comment:
This is now silently dropped, so CI would use pinned release instead of latest

Reply:
No, that variable is still respected; even after my installer PR.